### ISSUE TITLE
A11Y Issue - Default Images need an ALT Tag

### ISSUE THREAD

**Issue by aitala** [backdrop/backdrop-issues#7025]
## Description of the bug

Default images do not have a way to add ALT tags. This fails accessibility in SiteImprove.

## Steps To Reproduce

Set up a content type with an image field and provide a default image.  There is no means to add a default ALT tag to go with it.

## Actual behavior

No ALT tag field

## Expected behavior

There should be an additional text field for an ALT tag.

**Comment by indigoxela** [comment]
That's true - no way to add an alternative text for default images. They simply get an empty alt attribute.
While this may be correct sometimes (an empty alt indicates a decorative image without meaning in that context), it can be desirable to have the ability for admins to add this text.

Should probably be stored in the field instance settings json along with the image, I guess.

**Comment by aitala** [comment]
In our case the image is not decorative and should be pulled in an RSS news feed but occasionally no image is present, so we need the default image just in case.  Its getting flagged by Site Improve as an A11y issue as we're working to update to the new ADA requirements coming up in April 2026...

**Comment by indigoxela** [comment]
@aitala are you able to provide a pull request?

**Comment by aitala** [comment]
I'm not sure what you mean?
E

**Comment by indigoxela** [comment]
> I'm not sure what you mean?

If you have an idea, what the solution looks like, you could provide a pull request. No requirement, though.

**Comment by aitala** [comment]
> > I'm not sure what you mean?
> 
> If you have an idea, what the solution looks like, you could provide a pull request. No requirement, though.

Sorry, I dont have a good idea
